uptane-whitepaper.md

```diff
@@ -1 +1,107 @@
+- *Freeze attack:* continues to send the last known update to an ECU, even if a newer update exists.
+- *Partial bundle installation attack:* allows only part of an update to install by dropping traffic to selected ECUs.
+
+**Deny functionality:** This grouping ups the threat ante a bit further by causing vehicles to fail to function in one of the following ways:
+- *Rollback attack:* tricks an ECU into installing outdated software with known vulnerabilities.
+- *Endless data attack:* causes an ECU to crash by sending it an infinite amount of data until it runs out of storage.
+- *Mixed-bundles attack:* shuts down an ECU by causing it to install incompatible versions of software updates that must not be installed at the same time. Attackers can accomplish this by showing different bundles to different ECUs at the same time.
+- *Mix-and-match attack:*  causes an ECU to use arbitrary combinations of new versions of images.
+
+**Control:** The last and most severe threat is if an ECU can be forced to install software of the attacker’s choosing, thus ceding control of that unit. This means an attacker can arbitrarily modify the vehicle’s performance through an arbitrary software attack, in which the software on an ECU is overwritten with a malicious software program.
+
+To prevent these attacks, a secure over-the-air update system must do more than sign the contents of updates. While a cryptographic signature provides some protection against an arbitrary software attack, this strategy alone is not enough to protect against the other attacks enumerated above. In addition, the signing key itself is a single point of failure for the system. If an attacker gets control of this key, they have full control of all updatable ECUs.
+
+Instead, over-the-air updates require a solution that addresses all of the above attacks and is also compromise resilient. With compromise resilience, accessing control of a repository or a signing key is not sufficient to compromise the security of the entire system. In addition, compromise resilient systems have built-in mechanisms to recover from a compromise.
+
+## **What does Uptane do differently?**
+One of the strengths of Uptane is that it designed to work with existing systems rather than replacing such systems. It's a hallmark of this approach that Uptane never fights against best practices, but rather expands and/or improves the best practices of highly resilient software. It does this by adding a more realistic approach to it software update strategies. As stated at the beginning of this paper, Uptane acknowledges that compromise is not a matter of *if,* but of *when*. Hacks **will** occur and the best defense is a strategy that can isolate and limit exposure to attacks, and equally important, make it easier to recover from a compromise. The building blocks for this state rest on four design principles.
```

Comment:
```suggestion
One of the strengths of Uptane is that it designed to work with existing systems rather than replacing such systems. It's a hallmark of this approach that Uptane never fights against best practices, but rather expands and/or improves the best practices of highly resilient software. It does this by adding a more realistic approach to software update strategies. As stated above, Uptane acknowledges that compromise is not a matter of *if,* but of *when*. Hacks **will** occur and the best defense is a strategy that can isolate and limit exposure to attacks, and equally important, make it easier to recover from a compromise. The building blocks for this state rest on four design principles.
```